query/src/test/java/org/infinispan/query/queries/phrases/QueryPhrasesTest.java

```diff
@@ -46,8 +46,17 @@ public QueryPhrasesTest() {
       cleanup = CleanupPhase.AFTER_METHOD;
    }
 
+   @SuppressWarnings("unused")
```

Comment:
What is the purpose of this unused constructor?

Reply:
@anistor I'm using this constructor without argument on jdg-functional-tests wrapping QueryPhrasesTest.java